src/index.js

```diff
@@ -1,12 +1,12 @@
 var WalkManifest = require('./walk-manifest');
 var WriteData = require('./write-data');
 
-var main = function(options) {
+var main = function(options, callback) {
   console.log("Gathering Manifest data...");
-  var resources = WalkManifest(options.decrypt, options.output, options.input);
-
-  console.log("Downloading additional data...");
-  return WriteData(options.decrypt, options.concurrency, resources);
+  WalkManifest(options.decrypt, options.output, options.input, function(err, resources) {
+    console.log("Downloading additional data...");
+    callback(WriteData(options.decrypt, options.concurrency, resources));
```

Comment:
should callback be a node-style callback? Also, if `err` is an error object, we probably don't want to call `WriteData`.